manifests/config.pp

```diff
@@ -4,11 +4,22 @@
 #
 class uchiwa::config inherits uchiwa {
 
-  file { '/etc/sensu/uchiwa.json':
-    ensure  => file,
-    content => template('uchiwa/etc/sensu/uchiwa.json.erb'),
-    owner   => uchiwa,
-    group   => uchiwa,
-    mode    => '0440',
+
+  if count($uchiwa::users) > 0 {
```

Comment:
Please do not have two entrypoints for when users or set or not.
Reduce duplication here, just have one template